### PR TITLE
[keras/metrics] Standardise docstring usage of "Default to"

### PR DESCRIPTION
**Pull request by SamuelMarks**
This is one of many PRs. Discussion + request to split into multiple PRs @ #17748

### CODE REVIEW

File: keras/metrics/confusion_metrics.py [hunk 1]
```diff
@@ -36,11 +36,11 @@ class _ConfusionMatrixConditionCount(base_metric.Metric):
...
       confusion_matrix_cond: One of `metrics_utils.ConfusionMatrix` conditions.
-      thresholds: (Optional) Defaults to 0.5. A float value or a python
-        list/tuple of float threshold values in [0, 1]. A threshold is compared
-        with prediction values to determine the truth value of predictions
+      thresholds: (Optional) A float value or a python list/tuple of float
+        threshold values in [0, 1]. A threshold is compared with prediction
+        values to determine the truth value of predictions
         (i.e., above the threshold is `true`, below is `false`). One metric
-        value is generated for each threshold value.
+        value is generated for each threshold value. Defaults to 0.5.
```
Comment (qlzh727) [inline keras/metrics/confusion_metrics.py:43]: Please quote the 0.5 here.

File: keras/metrics/iou_metrics.py [hunk 2]
```diff
@@ -465,7 +467,7 @@ class MeanIoU(IoU):
       sparse_y_pred: Whether predictions are encoded using integers or
         dense floating point vectors. If `False`, the `tf.argmax` function
         will be used to determine each sample's most likely associated label.
-      axis: (Optional) Defaults to -1. The dimension containing the logits.
+      axis: (Optional) -1 dimension contains the logits. Defaults to `-1`.
```
Comment (qlzh727) [inline keras/metrics/iou_metrics.py:470]: This sentence seems broken. Please change it to:

The dimension contains the logits. Defaults to `-1`.

Same for all other places.

**Review by qlzh727 — CHANGES_REQUESTED** [review]

File: keras/metrics/confusion_metrics.py (hunk 1 above)
Comment (SamuelMarks) [reply keras/metrics/confusion_metrics.py:43]: Fixed

File: keras/metrics/iou_metrics.py (hunk 2 above)
Comment (SamuelMarks) [reply keras/metrics/iou_metrics.py:470]: Done.

```
fastmod -F -- '-1 dimension contains the logits. Defaults to `-1`.' 'The dimension containing the logits. Defaults to `-1`.'
```
